WordPress/Classes/ViewRelated/Pages/EditPageViewController.swift

```diff
@@ -10,14 +10,14 @@ class EditPageViewController: UIViewController {
     var onClose: (() -> Void)?
 
     convenience init(page: Page) {
-        self.init(page: page, blog: page.blog, postTitle: nil, content: nil, appliedTemplate: nil)
+        self.init(page: page, blog: page.blog, postTitle: nil, content: nil)
```

Comment:
This is one more step to consolidating `EditPostViewController` and `EditPageViewControleler`, which is great.